inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -189,6 +189,14 @@ def arxiv_plot_extract(obj, eng):
             current_app.logger.exception(err)
             delete_empty_key(obj, 'figures')
             return
+        except Exception as err:
```

Comment:
Think we should catch both errors as we handle them the same anyways..

`except (DelegateError, Exception) as err:`